pkg/plugin/linuxutil/netstat_stats_linux.go

```diff
@@ -22,6 +22,21 @@ const (
 	pathNetSnmp    = "/proc/net/snmp"
 )
 
+var (
+	nodeIP     = os.Getenv("NODE_IP")
+	ignoreList = []string{
+		"0.0.0.0",
+		"127.0.0.",
+	}
+)
+
+func init() {
```

Comment:
Who's calling this?

Reply:
@anubhabMajumdar The Go runtime does at package initialization time: https://go.dev/ref/spec#Package_initialization